bin/blade-icons-generate

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env php
+<?php
+
+use BladeUI\Icons\Generation\IconGenerator;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication as Command;
+
+require __DIR__.'/../vendor/autoload.php';
```

Comment:
This is less accurate at finding the autoloader than my original method was. I'm now getting errors because of this change, see:

```bash
± |experimental-generator U:1 ✗| → ./vendor/bin/blade-icons-generate

Warning: require(/Users/danpock/GitProjects/blade-boxicons/vendor/blade-ui-kit/blade-icons/bin/../vendor/autoload.php): Failed to open stream: No such file or directory in /Users/danpock/GitProjects/blade-boxicons/vendor/blade-ui-kit/blade-icons/bin/blade-icons-generate on line 9
```

Reply:
Did you run `composer install`?

Reply:
Yes. I did, but you maybe overlooking that this "bin" script shouldn't use the `vendor` folder in `blade-icons`. But should instead use the `vendor` folder from the project it's being required by. 

`__DIR__` is the directory of the file itself - so `blade-icons/bin` meaning the `/../` only ever gets us to `blade-icons/` and then to `vendor/`. In total, this transforms `blade-icons/bin` to `blade-icons/vendor/autoload.php`.

However when an end-user installs using composer composer this folder won't exist at all. So it should use that icon packs vendor folder since `blade-icons` will live there already. See here:
```
~/GitProjects/blade-boxicons
± |experimental-generator ✓| → ll vendor/blade-ui-kit/blade-icons/vendor
ls: vendor/blade-ui-kit/blade-icons/vendor: No such file or directory
```

Also - once the package is installed in a consumer icon pack the path will be different. So using `__DIR__` and multiple `/../` would require numbers traversing to parent diers. Since for instance in my `blade-boxicons` (`~/GitProjects/blade-boxicons`) package this code will exist at:
* `~/GitProjects/blade-boxicons/vendor/bin/blade-icons-generate` (symlink)
* `~/GitProjects/blade-boxicons/vendor/blade-ui-kit/blade-icons/bin/blade-icons-generate` (real)

But `__DIR__` always resolves to the true location; so in either case, the actual path would need to be `__DIR__ . '/../../../autoload.php`.

Reply:
TL;DR: The only reason I think this worked for you is because you did do `composer install` inside of your `blade-icons` folder. And you're using `path` instead of `vcs`, but once you switch to `vcs` (which is basically what composer does by default) this will break.

```
        {
            "type": "path",
            "url": "../blade-icons"
        }
```

So if you deleted the vendor folder from your `blade-icons` folder while you develop on `blade-heroicons` then it's more accurate to how composer will install things for real. Since normally `blade-heroicons/vendor/blade-ui-ki/blade-icons/vendor/` should (and will) never exist for the end users of this package.